### PR TITLE
python3 installer

### LINKED ISSUE #139: support new Python 3 installer (formerly in Perl)
@donsizemore this is the new branch I mentioned today: https://github.com/IQSS/dataverse/tree/3937-rewrite-installer-in-python-again

Here's the "Rewrite installer in Python 3" issue in the main repo: https://github.com/IQSS/dataverse/issues/3937

If would help us gain confidence in the new Python 3 installer if we could exercise it with our spin up scripts.

### PR DESCRIPTION
**Pull request by donsizemore**
closes #139

### CODE REVIEW

File: templates/interactive.config.j2 [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+DOI_PASSWORD = {{ dataverse.doi.password }}'
+DOI_BASEURL = {{ dataverse.doi.baseurl }}'
+DOI_MDCBASEURL = {{ dataverse.doi.mdcbaseurl }}'
+[comments]
+HOST_DNS_ADDRESS = :(enter numeric IP address, if FQDN is unavailable)
+GLASSFISH_USER = :This user will be running Glassfish service on your system.\n - If this is a dev. environment, this should be your own username; \n - In production, we suggest "glassfish" or another unprivileged user\n:
+GLASSFISH_DIRECTORY = 
+ADMIN_EMAIL = :\n(please enter a valid email address!) 
+MAIL_SERVER = 
+POSTGRES_SERVER =
```
Comment (pdurbin) [inline templates/interactive.config.j2:33]: I don't really understand what it means when there's no value? I assume it'll just use the default (localhost in this case, probably).

File: tasks/dataverse-build.yml [hunk 2]
```diff
@@ -59,7 +59,7 @@
     create: yes
 
 - name: build warfile with default tests. tail /tmp/dataverse/mvn.out for gory details.
-  shell: "mvn -T 2C package > mvn.out"
+  shell: "mvn -T 2C package -Dmaven.test.skip=true > mvn.out"
```
[the remark this answers is not in the document]
Comment (donsizemore) [reply tasks/dataverse-build.yml:62]: just so the build process runs cleanly =) whether to run tests is a group_var boolean, and if enabled the specified test string is also a group_var.

File: templates/interactive.config.j2 (hunk 1 above)
Comment (donsizemore) [reply templates/interactive.config.j2:33]: hmmm. i think this was a partial template that got included by mistake. i'll remove it.

File: tasks/dataverse-install.yml [hunk 3]
```diff
@@ -41,9 +40,7 @@
     dest: '{{ glassfish_dir }}/glassfish/lib/'
   when: jdbcurl | length > 0
 
-- name: copy jhove configs
-  shell: "cp /tmp/dvinstall/jhove* {{ glassfish_dir }}/glassfish/domains/{{ dataverse.glassfish.domain }}/config"
-
+# python3 installer still calls glassfish-setup.sh
```
[the remark this answers is not in the document]
Comment (donsizemore) [reply tasks/dataverse-install.yml:43]: i had been meaning to fix that.

**Pull request merged**